specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/locations.json

```diff
@@ -270,23 +275,35 @@
           "format": "int64"
         },
         "cores_available": {
-          "description": "The number of courses available in the region.",
+          "description": "The number of cores available in the region.",
           "type": "integer",
           "format": "int64"
         }
-      }
+      },
+      "readOnly": true
     },
     "QuotaCapability": {
       "description": "The regional quota capability.",
       "properties": {
+        "cores_used": {
+          "description": "The number of cores used in the subscription.",
+          "type": "integer",
+          "format": "int64"
```

Comment:
Would int32 make more sense size-wise?

Reply:
> Would int32 make more sense size-wise?

No, server data type is `long`

Reply:
I don't envy someone with a over a 2 billion core bill 😆